lib/simplifyExpression/collectAndCombineSearch/LikeTermCollector.js

```diff
@@ -220,6 +223,19 @@ function getTermsForCollectingMultiplication(node) {
 }
 
 // A helper function for getTermsForCollectingMultiplication
+// e.g. nthRoot(x, 2), append '2': nthRootNode to terms dictionary
```

Comment:
so the '2' is unrelated to the 2 in nthRoot(x, 2)? If so, change the example - I'm also confused like Lili xD

what do you mean by index? I agree more descriptive would be better.